language/pl-PL/com_users.ini

```
@@ -4,119 +4,159 @@
 ; Note : All ini files need to be saved as UTF-8
 
 COM_USERS_ACTIVATION_TOKEN_NOT_FOUND="Nie znaleziono kodu weryfikującego. Sprawdź, czy Twoje konto jest już aktywowane i spróbuj się zalogować ponownie."
-COM_USERS_CAPTCHA_LABEL="CAPTCHA"
+COM_USERS_CAPTCHA_LABEL="Captcha"
 COM_USERS_DATABASE_ERROR="Błąd podczas pobierania użytkownika z bazy danych: %s"
-COM_USERS_EDIT_PROFILE="Dostosuj profil"
+COM_USERS_EDIT_PROFILE="Edytuj profil"
```

Od 1.5 tłumaczymy "Edit" na "Dostosuj", "New" na "Utwórz". Tak jest we wszystkich nowszych polskich książkach. Dlatego opowiadam się za pozostawieniem, jak jest 
```suggestion
COM_USERS_EDIT_PROFILE="Dostosuj profil"
```